sklearn/cluster/optics_.py

```diff
@@ -88,17 +88,29 @@ class OPTICS(BaseEstimator, ClusterMixin):
 
     cluster_method : string, optional (default='dbscan')
```

Comment:
I would suggest to make `xi` the default, or not default at all.

IMHO, the `dbscan` extraction makes most sense when used repeatedly and not single-shot as with this parameter.

Reply:
Actually it seems that below, `xi` is the default?

Reply:
yes, it was a synchronization issue, fixed.